growth/too/tasks/tiles.py

```diff
@@ -301,6 +300,24 @@ def get_planned_observations(
                         corners[2] = corners_copy[3]
                         corners[3] = corners_copy[2]
 
+                    fields = models.Field.query.filter_by(telescope=telescope,
+                                                          ra=ra, dec=dec).all()
+                    if len(fields) == 0:
```

Comment:
In the interest of minimizing the number of database transactions, let's always create a new field here.